sys/include/net/gnrc/ipv6/netif.h

```diff
@@ -49,13 +50,11 @@ extern "C" {
 #endif
 
 /**
- * @brief   Default MTU
- *
- * @see <a href="https://tools.ietf.org/html/rfc2460#section-5">
- *          RFC 2460, section 5
- *      </a>
+ * @brief   Default MTU (must be at least @ref IPV6_MIN_MTU)
  */
-#define GNRC_IPV6_NETIF_DEFAULT_MTU (1280)
+#ifndef GNRC_IPV6_NETIF_DEFAULT_MTU
+#define GNRC_IPV6_NETIF_DEFAULT_MTU (IPV6_MIN_MTU)
```

Comment:
Do you mind to add an according comment? And why does it have to be at least `IPV6_MIN_MTU`?